### ISSUE TITLE
Bug: "Search and replace in selected active takes names" doesn't add selected item number at start or end

### ISSUE THREAD

**Issue by chords-chords** [X-Raym/REAPER-ReaScripts#21]
When you insert "/E" in the "Insert at start" or "Insert at end" boxes the script doesn't add selected item numbers but instead adds "/E".
It only works if I comment out the 3rd and 4th lines in the code block below (copied from the script):
`ins_start = ins_start_in:gsub("/E", tostring(i + 1))
ins_end = ins_end_in:gsub("/E", tostring(i + 1))  
ins_start = ins_start_in:gsub("/T", track_name)  
ins_end = ins_end_in:gsub("/T", track_name)`
That of course prevents adding the track name and isn't really a solution to this bug.

**Comment by X-Raym** [comment]
@chords-chords I'll take a look, meanwhile, just run the script two times, one for start and one for end
It is safe to assume that a user who want item number in both Start and End of naming is quite rate cause redundant ! 😆

**Comment by chords-chords** [comment]
Thanks.
However I think that you misunderstood what I was trying to say. The script wouldn't add the numbers to takes' names even if I inserted "/E" in just one of those boxes - meaning that the "/E" feature was not working at all. I had to comment out `ins_start_in:gsub("/T", track_name)` so I could add the numbers to the start of the name - that was my initial goal. When that worked I just tested the other box and realized that it's the same issue, so I tried commenting out the  `ins_end = ins_end_in:gsub("/T", track_name)` and then it worked.
Anyway, you're an awesome scripter/coder. Thanks for all of your hard and generous work!
